years_to_hours.rb

```diff
@@ -0,0 +1,26 @@
+def ask(question)
+	print question + " "
+	gets.chomp 
+end
+
+def leap(number)
+	number / 4
+end
+
+# asks the user for a number of years, and then prints out how many hours are in that many years.
+years = ask("Enter a number of years")   
+years = years.to_i 
+hours = (years * 365 + leap(years)) * 24
```

Comment:
Looks good. I think it's not asking you to calculate leap years. You might just want to do -
`hours = years * 365 * 24`

Reply:
I thought it is more accurate to include the leap year in my calculations.
Otherwise it isn't asking for leap years to be included.
Thank you.

Reply:
Yeah, that makes sense @umensa. Great work anyway!